templates/systemd/system.service.tpl

```diff
@@ -21,6 +21,9 @@ LimitSTACK=10485760
 
 User={{.User}}
 ExecStart={{.DeployDir}}/scripts/run_{{.ServiceName}}.sh
+{{- if eq .ServiceName "prometheus"}}
```

Comment:
surprised to find out that eq is a prefix match

Reply:
The `.ServiceName` is not the `prometheus-9100.service` form, the name is misleading you (I think we should blame @lucklove ). The correct field name should be `ComponentName`

Reply:
It is `prometheus-9100.service`.

I first thought `eq` must be a full match but found out its prefix match.

Reply:
@july2993 No, it's is `prometheus`, it's the component name and not service name.

Reply:
😳 all right.